lua/entities/gmod_wire_expression2/core/color.lua

```diff
@@ -42,38 +42,51 @@ e2function void entity:setColor(r,g,b)
 	if !IsValid(this) then return end
 	if !isOwner(self, this) then return end
 
-	this:SetColor(ColorClamp(Color(r,g,b,this:GetColor().a)))
+	local c = ColorClamp(Color(r,g,b,this:GetColor().a))
+	this:SetColor(c)
+	duplicator.StoreEntityModifier(this, "colour", { Color = c })
```

Comment:
If you ran:

```PHP
Prop:setColor(255, 255, 255, 128)
Prop:setColor(255, 255, 255)
```

and then duplicated that prop, I think it'd end up using its default rendermode whereas it should use `RENDERMODE_TRANSALPHA`. This line should add its current rendermode to the modifier data.